public/app/core/components/sidemenu/TopSection.tsx

```diff
@@ -22,7 +22,7 @@ const TopSection: FC<any> = () => {
     getLocationSrv().update({ query: { search: 'open' }, partial: true });
   };
   const updateDBaaS = async () => {
-    const { settings } = await getBackendSrv().post('http://localhost/v1/Settings/Get');
+    const { settings } = await getBackendSrv().post(`${window.location.origin}/v1/Settings/Get`);
```

Comment:
why not just this?

```suggestion
    const { settings } = await getBackendSrv().post(`/v1/Settings/Get`);
```

Reply:
Grafana backend is in different port than our backend (without adding the host that port is added automatically).
And to use just localhost we will have to update our cross-origin policies.

Reply:
Oh wow, that's messed up! Got it